test/codeu/chat/server/BasicControllerTest.java

```diff
@@ -127,4 +127,30 @@ public void testTokenizer() {
     assertEquals("how are you", t2.next());
     assertFalse(t2.hasNext());
   }
+  
+  public void testFollowUser() {
+    final User user = controller.newUser("user");
+    final User user1 = controller.newUser("user");
+    
+    User.follow(user, user1);
+    assertFalse("ERROR: Failed to follow user");
```

Comment:
assertFalse takes a condition, not an error string.  You need to be asserting that the follow relationship was correctly created.